src/panels/config/users/dialog-add-user.ts

```diff
@@ -0,0 +1,238 @@
+            class="name"
+            .label=${this.hass.localize("ui.panel.config.users.add_user.name")}
+            .value=${this._name}
+            required
+            auto-validate
+            autocapitalize="on"
+            error-message="Required"
+            @value-changed=${this._nameChanged}
+            @blur=${this._maybePopulateUsername}
+          ></paper-input>
+          <paper-input
+            class="username"
+            .label=${this.hass.localize(
+              "ui.panel.config.users.add_user.username"
+            )}
+            .value=${this._username}
+            required
+            auto-validate
+            autocapitalize="none"
+            @value-changed=${this._usernameChanged}
+            error-message="Required"
+          ></paper-input>
+          <paper-input
+            .label=${this.hass.localize(
+              "ui.panel.config.users.add_user.password"
+            )}
+            type="password"
+            .value=${this._password}
+            required
+            auto-validate
+            @value-changed=${this._passwordChanged}
+            error-message="Required"
+          ></paper-input>
+          <ha-paper-dropdown-menu
+            .label=${this.hass.localize("ui.panel.config.users.editor.group")}
+          >
+            <paper-listbox
+              slot="dropdown-content"
+              .selected=${this._group}
+              @iron-select=${this._handleGroupChange}
+              attr-for-selected="group-id"
+            >
+              ${GROUPS.map(
+                (groupId) => html`
+                  <paper-item group-id=${groupId}>
+                    ${this.hass.localize(`groups.${groupId}`)}
+                  </paper-item>
+                `
+              )}
+            </paper-listbox>
+          </ha-paper-dropdown-menu>
+          ${this._group === SYSTEM_GROUP_ID_USER
+            ? html`
+                <br />
+                The users group is a work in progress. The user will be unable
+                to administer the instance via the UI. We're still auditing all
+                management API endpoints to ensure that they correctly limit
+                access to administrators.
+              `
+            : ""}
+        </div>
+        <mwc-button
+          slot="secondaryAction"
+          @click="${this._close}"
+          .disabled=${this._loading}
+        >
+          ${this.hass!.localize("ui.common.cancel")}
+        </mwc-button>
+        ${this._loading
+          ? html`
+              <div slot="primaryAction" class="submit-spinner">
+                <paper-spinner active></paper-spinner>
+              </div>
+            `
+          : html`
+              <mwc-button
+                slot="primaryAction"
+                .disabled=${!this._name || !this._username || !this._password}
+                @click=${this._createUser}
+              >
+                ${this.hass.localize("ui.panel.config.users.add_user.create")}
```

Comment:
I feel like this should be "Save" instead of update. Dont we use save most other places?

Reply:
We use "Update" everywhere.

Reply:
Why do we use update for config options and save for cards?

Reply:
🤷‍♂